content/docs/formal-verification/_index.md

```diff
@@ -0,0 +1,137 @@
+abstractions that allows us to specify the high-level building blocks of cryptographic protocols without having to worry
+too much about how each cryptographic primitive is implemented. This also allows us to instantiate the protocol with any
+primitive that satisfies the given equations and still be sure that the security proofs hold. On the other hand, it
+means that we cannot reason about some things like brute-force or padding-oracle attacks in a straight-forward manner.
+(If we want to do this, we need to model our primitives in a way that allow us to express these types of attacks within
+the model. Depending on the security properties we are interested in, this may either add complexity, which often has
+adverse effects on proving time, or may sometimes be impossible within the symbolic model.)
+
+Symbolic verification tools typically model the network as untrusted, allowing the attacker to intercept, delay, modify,
+and replay messages between participants at will. This is known as the _Dolev-Yao model_, and was first described in the
+paper [_On the Security of Public-Key Protocols_](https://www.cs.huji.ac.il/~dolev/pubs/dolev-yao-ieee-01056650.pdf).
+Individual tools often provide abstractions (like the `reliable-channel` builtin in Tamarin or the `passive` keyword in
+Verifpal) that can be used to restrict the capabilities of the attacker in different ways.
+
+Tools like [CryptoVerif](https://bblanche.gitlabpages.inria.fr/CryptoVerif/) analyze protocols in the _computational
+model_. Here, individual messages are modeled as bitstrings, cryptographic primitives are modeled as functions on
+bitstrings, and the adversary is modeled as an arbitrary probabilistic polynomial-time Turing Machine. The probability
+of the attacker successfully breaking the security of a given primitive has to be provided up-front by the user. The
+tool can then automatically prove indistinguishability between sequences of games (up to some probability), where the
+first game captures the analyzed protocol, and the final game is one where the required security property is obvious.
+The output is given as a bound on the probability of a successful attack as a function of the security parameters of the
+individual primitives used, and the number of protocol executions. A key benefit of the computational model is that this
+mimics how cryptographers usually prove security of cryptographic protocols. This makes it easier to carry over
+constructions from traditional pen-and-paper proofs to automated proofs.
+
+The computational model is clearly more expressive than the symbolic model. However, it is also presupposes a deep
+understanding of provable security and game-based proofs, which is not required when working in the symbolic model.
+
+Ultimately, which tool to use depends as much on your background as on the protocol you are trying to analyze. If you
```

Comment:
Should we consider framing them as complementary? I probably need to re-read again, but currently, it reads as if the tools are, to some extent, opposites.
Maybe we can point out that (as already done now) thatfor most users, especially those with little cryptographic analysis background, symbolic tools will provide the most value very fast. On the other hand, one could do everything in the computational model, but that’s painful. Power users like Inria tend to do tons of symbolism first and conclude the analysis with computational.